### PR TITLE
FINERACT-802-restrict-client-to-have-single-self-service-user

### PR DESCRIPTION
**Pull request by aimanoj**
## Description
Error message added for the above fineract issue

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Comment by edcable** [comment]
@fynmanoj Can you address the merge conflict so we can merge this for upcoming release?

**Comment by fynmanoj** [comment]
merge conflict resolved 
@edcable

File: app/global-translations/locale-en.json [hunk 1]
```diff
@@ -4230,11 +4230,10 @@
   "label.button.activateselfservice": "Activate Self Service User",
   "label.button.selfserviceactivatehelp": "Activate Self Service User Help",
   "label.createdselfserviceuser": "Self Service User created successfully",
-
   "label.input.maturity.instruction" :  "Maturity Instruction",
   "label.input.apply.maturity.instructions" :"Add Maturity Instructions",
   "label.input.transfer.to.account" : "Transfer To Savings Account",
   "validation.msg.fixeddepositaccount.transferToSavingsId.cannot.be.blank" : "Savings account for transfer cannot be blank.",
-
+  "error.msg.user.self.service.user.already.exist" : "Self Service User Id is already created. Go to Admin->Users to edit or delete the self-service user.",
```
Comment (luckyman20) [inline app/global-translations/locale-en.json]:
```suggestion
  "error.msg.user.self.service.user.already.exist" : "Self Service User ID is already created. Go to Admin -> Users to edit or delete the self-service user.",
```

**Review by luckyman20 — CHANGES_REQUESTED** [review]
@fynmanoj There's still a merge conflict. Can you please resolve it so we can merge your Pull Request?

File: app/global-translations/locale-en.json (hunk 1 above)
Comment (fynmanoj) [reply app/global-translations/locale-en.json]: done!

**Review by luckyman20 — APPROVED** [review]
Looks good, thanks @aimanoj!

**Comment by luckyman20** [comment]
@aimanoj @fynmanoj It still has a conflict. Can you please quickly fix it so that we can merge this?

**Comment by fynmanoj** [comment]
> @aimanoj @fynmanoj It still has a conflict. Can you please quickly fix it so that we can merge this?

done!

**Comment by luckyman20** [comment]
@fynmanoj Thanks for the contribution!

**Pull request merged**